tests/unittests/OperatorGradTest.cpp

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "BackendTestUtils.h"
+
+#include "glow/Base/Tensor.h"
+#include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/Graph/Graph.h"
+#include "glow/Graph/Nodes.h"
+#include "glow/IR/IR.h"
+#include "glow/IR/IRBuilder.h"
+#include "glow/IR/Instrs.h"
+
+#include "gtest/gtest.h"
+
+using namespace glow;
+
+class OperatorGradTest : public BackendTest {
+protected:
+  /// Compute gradients of placeholders in bindings_. Given
+  /// outputNode, representing F(Vars) = q, this function creates an L2 node
+  /// computing its MSE against zeros, G(Vars) = 0.5 * /F(Vars)^2 and computes
+  /// dG/dVar with respect to all variables in "main" function.
+  ///
+  /// Note that dG/dF = F(Vars), the same as outputNode's forward value.
+  ///
+  /// \param outputNode Node that contains result of f(x).
+  VariableGradientsList computeVarGrads(Node *outputNode) {
+    // A RegressionNode is created just to generate a fake gradient with the
```

Comment:
I suggest to remove this comment, because we already explain purpose of RegressionNode in comment above.